test_bst.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import pytest
+import bst
+
+
+@pytest.fixture(scope="function")
+def initial_node():
+    initial_node = bst.Node(10)
+    return initial_node
+
+
+@pytest.fixture(scope="function")
+def built_tree():
+    tree_root = bst.Node(10)
+    vals = [5, 15, 3, 7, 12, 20, 2, 4, 6, 9, 11, 14, 18, 22]
+    for val in vals:
+        tree_root.insert(val)
+    return tree_root
```

Comment:
even though it's not in the specs, I made a "populate" method for my node so that I could build a tree from a list without needing to first make a single node.